### DISCUSSION TITLE
Is there any plans to move away from using CSS-in-JS libraries?

### DISCUSSION THREAD

**Discussion by TKDev7** [mantinedev/mantine#3814] [Feature requests]
Is there any plans to move away from using CSS-in-JS libraries?

**Comment by vitalijalbu** [comment]
Would be fine to have 50% css and 50% js, becaus efor example I have css in js

**Comment by rtivital** [comment]
See https://github.com/mantinedev/mantine/discussions/3579#discussioncomment-5047179
